tests/src/testmaterial.cpp

```diff
@@ -143,8 +156,58 @@ auto testMaterial(Material mat) {
   return t;
 }
 
+template <StrainTags strainTag, typename MaterialImpl>
+auto testPlaneStrainAgainstPlaneStress(const double tol = 1e-10) {
+  TestSuite t(MaterialImpl::name() + " InputStrainMeasure: " + toString(strainTag));
+  std::cout << "TestPlaneStrinAgainstPlaneStress: " << t.name() << " started\n";
+
+  Eigen::Matrix3d e;
+  e.setRandom();
+
+  transformStrainAccordingToStrain<strainTag>(e);
+
+  // instantiate material models
+  LamesFirstParameterAndShearModulus matPar{.lambda = 0, .mu = 1000}; // nu = 0
```

Comment:
Can you also add the sanity check for `nu != 0` that some quantities should differ?